.gitignore

```diff
@@ -46,3 +46,4 @@ config/environments/*.local.yml
 
 # Avatars and such
 public/uploads
+.DS_Store
```

Comment:
This should be in your global ignore https://help.github.com/articles/ignoring-files/#create-a-global-gitignore  as this is specific only to certain operating systems and it not part of the project.

Reply:
no problem